repository.json

```diff
@@ -284,6 +284,23 @@
 				}
 			]
 		},
+		{
+			"details": "https://github.com/sublimelsp/LSP-Grammarly",
+			"name": "LSP-Grammarly",
+			"labels": [
+				"linting"
+			],
+			"releases": [
+				{
+					"sublime_text": ">=4070",
+					"platforms": [
+						"linux",
+						"osx"
+					],
```

Comment:
Windows is not supported by the underlying server?

Reply:
Probably the server runs fine on Windows, but 1) I have only tried it on Linux and Mac OS, 2) there were some (stale, invalid, and/or fixed) [issues](https://github.com/znck/grammarly/issues?q=is%3Aissue+windows) reported in the past.

Reply:
I've tried it on Windows now and it works fine.

Also didn't really see any relevant issues when looking at the list.

Reply:
Perfect! Thanks! I removed the restriction.